WebApp/src/components/BasicTest/BasicTestContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import './BasicTestContainer.css';
+import { IonText, IonButton } from '@ionic/react';
+import React from 'react';
```

Comment:
On peut importer useState directement.
```suggestion
import React, { useState } from 'react';
```